azurelinuxagent/ga/exthandlers.py

```diff
@@ -319,22 +319,13 @@ def handle_ext_handler(self, ext_handler, etag):
 
         try:
             state = ext_handler.properties.state
-            max_retries = 5
-            retry_count = 0
-            while ext_handler_i.decide_version(target_state=state) is None:
-                if retry_count >= max_retries:
-                    err_msg = "Unable to find manifest for extension {0}".format(ext_handler_i.ext_handler.name)
-                    ext_handler_i.set_operation(WALAEventOperation.Download)
-                    ext_handler_i.set_handler_status(message=ustr(err_msg), code=-1)
-                    ext_handler_i.report_event(message=ustr(err_msg), is_success=False)
-                    return
-                time.sleep(2**retry_count * 10)
-                retry_count += 1
-            if retry_count != 0:
+            if ext_handler_i.decide_version(target_state=state) is None:
+                err_msg = "Unable to find manifest for extension {0}".format(ext_handler_i.ext_handler.name)
```

Comment:
Re-reading the error message that I wrote for this, I realize that it is inaccurate. If the agent can't find the manifest, it will throw an exception.

Maybe something like this?
```python
"Unable to find version {0} in manifest for extension {1}".format(ext_handler_i.ext_handler.properties.version, ext_handler_i.ext_handler.name)
```

Reply:
Sure, will update